code/modules/antagonists/abductor/equipment/glands/heal.dm

```diff
@@ -0,0 +1,178 @@
+/obj/item/organ/heart/gland/heal
+	true_name = "organic replicator"
```

Comment:
I think  you can do better for a pointless science name than this.

What about "autotomy/genesis marrow".

That's _autotomy_, the term for when lizard tails fall off, not _autonomy_.